iChef-WebApi/src/test/java/com/kitchen/iChef/Service/Hashing/BCryptPasswordEncoderTest.java

```diff
@@ -0,0 +1,33 @@
+package com.kitchen.iChef.Service.Hashing;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+public class BCryptPasswordEncoderTest {
+    @Test
+    @DisplayName("Test match Success Case")
+    void match_success()
+    {
+        String password = "popcorn";
```

Comment:
We should use var instead of the type of the variable when possible. It makes refactoring code easier